app/models/check.rb

```diff
@@ -0,0 +1,48 @@
+class Check < ActiveRecord::Base
+  belongs_to :card
+
+  DELAYS = [12, 3*24, 7*24, 14*24, 31*24]
+
+  NUMBER_OF_FAILS = 2
+
+  def mark_correct!
+    increment_wins
+    self.set_review_date
+    self.save
+  end
+
+  def mark_incorrect!
+    if self.losing_streak < NUMBER_OF_FAILS
+      increment_losses
+    else
+      reset_losses
+    end
+    self.set_review_date
+    self.save!
+  end
+
+  protected
+
+  def increment_losses
+    self.times_reviewed += 1
+    self.losing_streak  += 1
+  end
+
+  def increment_wins
+    if !self.times_reviewed.zero?
+      self.box          += 1
```

Comment:
Unnecessary spacing detected.<br>Operator `+=` should be surrounded with a single space.